### ISSUE TITLE
Deployment information relevant for external users?

### ISSUE THREAD

**Issue by peterdesmet** [inbo/fish-tracking-data-publication#5]
As a test I've made a deployment (+ receiver) extraction from the data.

https://github.com/inbo/fish-tracking-data-publication/blob/master/data/output/temp/deployment.csv

Each record is basically a deployment (file) that was uploaded to the database. I wonder if that information will be relevant for the open data/external users ?

1. Since the coordinates of the receivers are going to be in the detections, they already know **where** receivers are deployed.
2. The information that is not in the detections is the **from when till when** a receiver was deployed, which might give an indication of where animals _could_ be detected, but weren't (some kind of absence data). Is that kind of use to be trusted? Do any of you use this as such?
3. Also, I don't actually have the time window in the data: `deployment.deploy_data_time` is present, but `deployment.recover_date_time` is not. Is it in the database?
4. There is also quite a lot of [coordinates information](raw_deployment_deploy_lat) attached to a deployment: `deployment.deploy_lat/long`, `deployment.intended_lat/long` and `deployment_recover_lat/long`, but only `deployment.deploy_lat/long` is consistently populated, so I've used that as the location of the fish in the detections. Are the other coordinates used?

**Comment by PieterjanVerhelst** [comment]
1. To my opinion, it is necessary to give the locations of the stations, since this is the added value of an international network. If they are already given with the detections, than it is no problem to give them from the network
2. we don't us it as such, but as it is a dynamic network, so relevant to mention when stations were active. For example, if I want to know if Belgian fish migrated to The Netherlands, I want to know if there were active stations in the first place (for the period of migration). However, should we give the timestamps for every deployment or make a separate list of when specific stations were not active?
3.  Yes, this is in the database. 
4. Indeed, `deployment.deploy_lat/long` is the most consistent to use.

**Comment by jreubens** [comment]
Point 1 and 3 PJ covered.
For Point 2: we don't use as such, but have been thinking to use it. However, For the moment we don't have a good way to do so. Something to think about...
For Point 4: only deploylat/long are consistently filled out. The others can be used. This can be interesting for instance if you work with a mooring in a dynamic environment where you expect moorings to move around... (e.g. in wind farms). however in most occasions this is redundant as stations are fixed (e.g. along a river bank).

**Comment by peterdesmet** [comment]
Thanks! So to provide information on the **past** status of the network, we could provide a file with **deployments** (where even a recover and immediate redeployment will be seen as two deployments):

* [x] `deployment_fk`: unique key for the deployment
* [x] `deployment_station_name`: coded and stable name of the location. **Allows linking to the detections**
* [x] `deployment_deploy_date_time`: start of deployment
* [x] `deployment_recover_date_time`: end of deployment
* [x] `deployment_deploy_lat`
* [x] `deployment_deploy_long`
* [ ] `file`: **maybe**, not sure how relevant this is?
* [ ] `receiver`: code of the receiver there at that time. Is combination of `receiver_model_number` and `receiver_serial_number` so not relevant to provide these fields separately
* [x] `receiver_owner_organization`: relevant for contacting people
* [ ] `receiver_status`: I assume this is the status of the receiver AFTER deployment? So, filtering on this gives an indication of the current status of the network (with a start date_time shortly after `deployment_recover_date_time` and an open end date?

@PieterjanVerhelst @jreubens can you let me know about the relevance and questions regarding the above fields?

Additional question: how is the system aware of deployments with no detections? Is an empty file uploaded or is it managed manually in the DB? In any case, I think it's important to also list these in the deployment export.

**Comment by PieterjanVerhelst** [comment]
I am not sure if the file needs to be given. Or is this 'read onlye'? Do external users get the data via a data download?
Also, to my opinion it is not necessary to give the receiver; station with coordinates suffices.
The owner organization is relevant I think: if a user wants more information about one particular station, they know who to contact.
To your additional question: an empty file is uploaded

**Comment by peterdesmet** [comment]
How to provide this information is to be decided, but it should be public (open), simple, downloadable and read-only. Ideally it's also versioned (so you can find an older version) and findable under a stable URL. An easy implementation would be to generate such a CSV, place it on GitHub (or even better: Zenodo) each time we republish the data. For a more "live" version, we could guide users to the database (provided they can access that information publicly, without login).

**Comment by PieterjanVerhelst** [comment]
I think I misunderstood, does the `file` in the checkbox stand for a detection file per deployment?

**Comment by peterdesmet** [comment]
Yes, e.g. `VR2W_120095_20150609_1.csv`

**Comment by PieterjanVerhelst** [comment]
Ok, than I didn't misunderstood. However, on second thoughts it may be interesting to include the file. For example when someone is interested if his/her tagged fished were detected at a particular station within a particular time window.

**Comment by peterdesmet** [comment]
How would you know that from the file name? If you have all the detections in one database (with open receivers), you would know that anyway, right? And for the open data detections, you also have the station_name in the detections.

**Comment by PieterjanVerhelst** [comment]
Indeed, true. And since this is about the status of the network, we don't need to take into account the separate files.

**Comment by peterdesmet** [comment]
Well, the file name itself is not relevant, but filename = deployment, and those are relevant to know the **historic** status of the network. E.g.

station | deployment | deployed at | recovered at | status
--- | --- | --- | --- | ---
A | 1 | 2016-01-01 | 2016-06-30 | -
A | 2 | 2016-09-30 | 2017-01-01 | -
A | 3 | 2017-01-01 | 2017-09-01 | ACTIVE

1. You can see there was a gap in the detection window between 2016-06-30 and 2016-09-30 (1 and 2)... so no fishes could be detected then.
2. You can see station A is still active... and assume since 2017-09-01.
3. You could filter on status =  `ACTIVE` to get all the currently active receivers.

**Comment by PieterjanVerhelst** [comment]
Yes indeed, that is relevant to keep track of the activity of a station
